icons/font-size-higher.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m4 18 6-12 6 12" />
+  <path d="M5.5 15h9M16 8l2-2 2 2" />

```

Comment:
What about something like ![baseline (2)](https://user-images.githubusercontent.com/25524993/236631719-e7f078df-597c-4960-a573-ae45dfcc3fac.svg) or ![baseline](https://user-images.githubusercontent.com/25524993/236631720-a4c5cad1-a072-490e-81a9-5c051172c912.svg)?

Reply:
I prefer the second one as it looks less cluttered

Reply:
Another option so it doesn't look like the `^` character ![baseline (3)](https://user-images.githubusercontent.com/25524993/236633202-a781d402-c280-4cae-878d-7cd896d6177d.svg)

Reply:
Good point - I'm not sure about having two chevrons though. Maybe we could go with the one with the arrow, and also do that for the `font-size-lower` icon?